### ISSUE TITLE
`yarn run dev` doesn't launch the dev server and just quits without any errors.

### ISSUE THREAD

**Issue by prajjwaldimri** [SimulatedGREG/electron-vue#669]
##### Describe the issue / bug.

The dev runner is failing since yesterday. I tried reverting to old commits, creating a fresh project and copying over the code but it never launches the dev server on my pc. (Although, weirdly it works on my mate's pc. He uses Ubuntu and I use Arch) 

##### If visual, provide a screenshot.

![image](https://user-images.githubusercontent.com/9110203/43730534-4880781c-99c9-11e8-9549-506a9021b749.png)
![image](https://user-images.githubusercontent.com/9110203/43730525-3f98404a-99c9-11e8-8ba6-6b7bea770684.png)
![image](https://user-images.githubusercontent.com/9110203/43730519-34142432-99c9-11e8-8bc5-423e438498a1.png)

**The script just quits at this point. No errors whatsoever.**

##### Tell me about your development environment.
  * Node version: 10.7.0
  * Yarn version: 1.9.4
  * vue-cli version: (if necessary)
  * Operating System: Arch Linux

**Comment by diegoflorez** [comment]
Same is happening to me

**Comment by diegoflorez** [comment]
It's a glibc 2.28 related issue.
I have solved this in ArchLinux downgrading glibc and lib32-glibc to 2.27-3
`sudo pacman -U /var/cache/pacman/pkg/lib32-glibc-2.27-3-x86_64.pkg.tar.xz /var/cache/pacman/pkg/glibc-2.27-3-x86_64.pkg.tar.xz`

**Comment by prajjwaldimri** [comment]
@diegoflorez Yup, multiple apps like Mailspring and Slack are also broken on Arch due to glibc-2.27. I faced this error on the [Electron 2.0 PR](https://github.com/SimulatedGREG/electron-vue/issues/637), and when I reverted back to the original electron-vue boilerplate it worked successfully even without downgrading glibc. I guess this is a electron > 2.0 issue.

For anyone who wants more details on this bug, it is being discussed here: https://bugs.archlinux.org/task/59550
